CODEOWNERS.txt

```diff
@@ -0,0 +1,14 @@
+# Where component owners are known, add them here.
+
+/oneTBB/src/tbb/ @pavelkumbrasev
+/oneTBB/src/tbb/ @dnmokhov
+/oneTBB/src/tbb/ @JhaShweta1
+/oneTBB/src/tbb/ @sarathnandu
+/oneTBB/src/tbbbind/ @isaevil
```

Comment:
What about `/oneTBB/include/`, `/oneTBB/integration`, `/oneTBB/examples/`, `/oneTBB/doc/`?

Reply:
@sarathnandu For doc/ please add @aepanchi

Reply:
> What about `/oneTBB/include/`, `/oneTBB/integration`, `/oneTBB/examples/`, `/oneTBB/doc/`?

The maintainers are specified component wise. Since these folders and files can be associated with a component, I didn't mention specific maintainers.

Reply:
> @sarathnandu For doc/ please add @aepanchi

Done.